kalite/distributed/templates/distributed/base.html

```diff
@@ -182,8 +182,8 @@
                               <form class="navbar-form" action="{% url 'search' %}" method="get" id="search-box" role="search">
                                 {% comment %}translators: this will appear in the navigation bar. please try to keep it as short as possible.{% endcomment %}
                                 <div class="input-group">
-                                  <input type="text" name="query" id="search" class="form-control" placeholder="{% trans 'Topic, video, exercise...' %}" />
-                                  <button id="search-button" class="btn btn-default" type="submit"><i class="glyphicon glyphicon-search"></i></button>
+                                  <input type="text" name="query" id="search" class="form-control" title="Type search term here" placeholder="{% trans 'Topic, video, exercise...' %}" />
+                                  <button id="search-button" class="btn btn-default" type="submit" value="Search"><i class="glyphicon glyphicon-search"></i></button>
```

Comment:
@rtibbles, sorry for my late reply, messy days at uni.

Strictly speaking, 'value' attribute may not be the requirement here as it would be for input field with type="submit". However, screen readers tend to have similar issues as browsers: even though specification says one thing, they may render speech with slight differences, and in the absence of descriptive text labels they rely on "surroundings" to infer (aka guess ;) the meaning and purpose of control elements. 

Take for example the following block of code:

```html
<input title="Type your search term here" placeholder="Topic, video, exercise..." type="text">
<button type="submit" value="Search"><i class="glyphicon glyphicon-search"></i></button>
```
It gives these speech outputs in 3 different screen readers:

**JAWS**

Topic video exercise dot dot dot 
edit Type your search term here 
type and text
button to activate press space bar

**NVDA**

Topic video exercise
edit Type your search term here
blank
button

**ZoomText**

Topic video exercise
edit box
button

Similarly, in our full code with ARIA roles implemented, JAWS reads "search landmark region 
Topic video exercise dot dot dot" when input field is in focus, while NVDA reads "search landmark button" when the search button is in focus, ZoomText doesn't seem to see neither ARIA roles nor title attribute, and other screen readers may render something completely different.

To sum up: even though not strictly required, value="Search" in the button element may give additional inferential cues to some screen readers in the absence of descriptive text label.

Hope this helps... :)